CMakeLists.txt

```diff
@@ -28,12 +28,12 @@ endif()
 
 #################### External dependencies ####################
 
-find_package(Libxc QUIET)
+find_package(Libxc 6.0.0 REQUIRED)
```

Comment:
That may not work as expected (I don't understand from the CMake documentation, whether this would accept everything above 6.0.0). I'd suggest not to specify the version in the `find_package()` call and have a 
```
if (... VERSION_LESS "6.0.0")
  message(ERROR "....")
endif()
```

Reply:
I have decided to implement the manual check, because specifying the lower limit only does not work as intended.